httpbp/client_middlewares.go

```diff
@@ -349,3 +354,37 @@ func PrometheusClientMetrics(serverSlug string) ClientMiddleware {
 		})
 	}
 }
+
+func (c clientFaultMiddleware) Middleware() ClientMiddleware {
+	return func(next http.RoundTripper) http.RoundTripper {
+		return roundTripperFunc(func(req *http.Request) (*http.Response, error) {
+			resumeFn := func() (*http.Response, error) {
+				return next.RoundTrip(req)
+			}
+
+			faultFn := func(code int, message string) (*http.Response, error) {
```

Comment:
```suggestion
			resume := func() (*http.Response, error) {
				return next.RoundTrip(req)
			}

			fault := func(code int, message string) (*http.Response, error) {
```

I would try to avoid suffixing it with the type for the same reason we don't suffix every variable with the type :slightly_smiling_face:

Reply:
Done!